keyboards/40percentclub/gherkin/keymaps/bbaserdem/keymap.c

```diff
@@ -21,16 +24,12 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     )
 };
 
-void matrix_init_user(void) {
-    // Set LED's to max
-    _delay_us(300);
-    backlight_config.level = 2;
-    backlight_config.enable = 1;
-    eeconfig_update_backlight(backlight_config.raw);
-    backlight_set(backlight_config.level);
+void matrix_init_keymap(void) {
```

Comment:
If you moved this to `keyboard_post_init_*`, you could use the noeeprom versions of these functions, and it would work properly